src/i18n/es.js

```diff
@@ -25,9 +25,9 @@ const subjectBlockedAccountInvalidPassword = `?subject=Cuenta%20bloqueada%20por%
 const urlControlPanel = `${urlDopplerLegacy}/ControlPanel`;
 const urlBuyMonthly = `${urlControlPanel}/AccountPreferences/UpgradeAccount?Plan=monthly`;
 const urlSiteTracking = `${urlControlPanel}/CampaignsPreferences/SiteTrackingSettings`;
-const urlSiteFromSignup = `${urlSite}/`;
-const urlSiteFromLogin = `${urlSite}/`;
-const urlSiteFromForgot = `${urlSite}/`;
+const urlSiteFromSignup = `${urlSite}/?utm_source=app&utm_medium=landing&utm_campaign=signup`;
+const urlSiteFromLogin = `${urlSite}/?utm_source=app&utm_medium=landing&utm_campaign=login`;
+const urlSiteFromForgot = `${urlSite}/?utm_source=app&utm_medium=landing&utm_campaign=restablecimiento-contrasenia`;
```

Comment:
Is it fine?

Reply:
I got these changes when I did the rebase

Reply:
Mmm, probably you have resolved conflicts in a wrong way.

Reply:
We've removed all utm parameters from this site, this shouldn`t be as a change.

Reply:
ok I'm going to check what could have happened